adding type AuthGitlab

shivpathak opened the pull request:
adding missing part of OAuth type GitLab.

pb82 commented:
@shivpathak the change works fine, any particular reason for removing the autogenerated code here: https://github.com/integr8ly/grafana-operator/pull/128/commits/ade249692cf4539f68607258974ba145e950b8fe#diff-6cbfee6e2db2fb9965c7e5cae99ad4c6L136 ?

After rerunning `operator-sdk generate k8s` it re-adds that code.

shivpathak commented:
> @shivpathak the change works fine, any particular reason for removing the autogenerated code here: [ade2496#diff-6cbfee6e2db2fb9965c7e5cae99ad4c6L136](https://github.com/integr8ly/grafana-operator/commit/ade249692cf4539f68607258974ba145e950b8fe#diff-6cbfee6e2db2fb9965c7e5cae99ad4c6L136) ?
> 
> After rerunning `operator-sdk generate k8s` it re-adds that code.

@pb82  Thanks you so much for reviewing the PR, actually that's autogenerated that's why I removed, previously I added manually but later realised it's automatically generated so removed back my manual changes.

pb82 commented:
Yes, please add it back (you can generate the code by running `operator-sdk generate k8s`) and then this is ready to be merged.

It's not really part of the build process, we try to include the auto generated code for the API in the repo.

shivpathak commented:
> Yes, please add it back (you can generate the code by running `operator-sdk generate k8s`) and then this is ready to be merged.
> 
> It's not really part of the build process, we try to include the auto generated code for the API in the repo.

@pb82 Okay.. doing it right now.. allow me 5 mins.. Thanks.

shivpathak commented:
> > Yes, please add it back (you can generate the code by running `operator-sdk generate k8s`) and then this is ready to be merged.
> > It's not really part of the build process, we try to include the auto generated code for the API in the repo.
> 
> @pb82 Okay.. doing it right now.. allow me 5 mins.. Thanks.

@pb82 Done.

pb82 commented:
perfect, thanks :+1:
